compute/instances/reference-content/understanding-qemu-guest-agent.mdx

```diff
@@ -0,0 +1,112 @@
+---
+meta:
+  title: Understanding the QEMU Guest Agent in Scaleway Instances
+  description: Discover how Scaleway Instance pricing works, including hourly rates, billing, and tips to manage costs effectively.
```

Comment:
I suspect this needs to be adjusted to match the document.

Reply:
Indeed, I forgot to update it. Done now.